sdk/tables/azure-data-tables/inc/azure/data/tables/tables_audience.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "dll_import_export.hpp"
+
+#include <azure/core/internal/extendable_enumeration.hpp>
+namespace Azure { namespace Data { namespace Tables {
+  /**
+   * @brief Audiences available for table service
+   *
+   */
+  class TablesAudience final {

```

Comment:
Re: singular vs plural - FWIW, in .NET they do name it BlobAudience and QueueAudience for Blob**s** and Queue**s** library: https://github.com/search?q=repo%3AAzure%2Fazure-sdk-for-net%20audience&type=code

Can't find Table(s)Audience in other SDKs yet, maybe because everyone is implementing this feature this month? Let's try to align with the other languages SDKs then - how do they name it?, although exact mapping does not always make sense, and we should also look if naming makes sense in context of C++ SDK (precedents in this library so far, precedents in other C++ SDK libraries).